themes/base.theme.bash

```diff
@@ -16,8 +16,12 @@ SCM_THEME_BRANCH_GONE_PREFIX=' ⇢ '
 SCM_THEME_CURRENT_USER_PREFFIX=' ☺︎ '
 SCM_THEME_CURRENT_USER_SUFFIX=''
 
-CLOCK_CHAR='☆'
-THEME_CLOCK_CHECK=${THEME_CLOCK_CHECK:=true}
+CLOCK_CHAR='⌚'
+THEME_CLOCK_CHAR_COLOR=${THEME_CLOCK_CHAR_COLOR:="${red}"}
+THEME_SHOW_CLOCK=${THEME_CLOCK_CHECK:-true}
```

Comment:
This is only taking the old variable into account. If a user defines `$THEME_SHOW_CLOCK`, it is not used at the moment, right? I think you'll have to check for both variables if you want to stay backwards compatible and want to use the new one (`$THEME_SHOW_CLOCK`) as well.